pom.xml

```diff
@@ -802,12 +808,12 @@
         <version.org.wso2.orbit.javax.xml.bind>2.3.1.wso2v1</version.org.wso2.orbit.javax.xml.bind>
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
-        <identity.governance.version>1.10.6</identity.governance.version>
+        <identity.governance.version>1.11.4-SNAPSHOT</identity.governance.version>
         <carbon.identity.framework.version>7.5.0</carbon.identity.framework.version>
         <maven.findbugsplugin.version>3.0.5</maven.findbugsplugin.version>
         <findsecbugs-plugin.version>1.12.0</findsecbugs-plugin.version>
         <maven.checkstyleplugin.excludes>**/gen/**/*</maven.checkstyleplugin.excludes>
-        <identity.event.handler.version>1.8.19</identity.event.handler.version>
+        <identity.event.handler.version>1.9.5-SNAPSHOT</identity.event.handler.version>
```

Comment:
The snapshot versions need to be bumped after dependent PRs are merged. The snapshot values will be replaced with new release versions.